python/ray/tune/tune.py

```diff
@@ -277,12 +281,35 @@ def run(run_or_experiment,
     Raises:
         TuneError: Any trials failed and `raise_on_failed_trial` is True.
     """
+    if global_checkpoint_period:
+        raise ValueError("global_checkpoint_period is deprecated. "
+                         "Set 'TUNE_GLOBAL_CHECKPOINT_S' instead.")
+    if queue_trials:
+        raise ValueError(
+            "queue_trials is deprecated. "
+            "Set 'TUNE_DISABLE_QUEUE_TRIALS=1' instead to disable queuing"
```

Comment:
Might be a nit, but it would be good to explicitly say tht TUNE_DISABLE_QUEUE_TRIALS is an environment variable